### PR TITLE
Added Github Gist

### PR DESCRIPTION
**Pull request by Ki-er**
Added Github Gist to Services

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -77,10 +77,11 @@ Freeside welcome contributions to these lists from staff and students at the Uni
 7. [#HullCSS Discord](https://hullcss.org/discord).
 8. [Hull Gaming Society Discord](https://discord.gg/Rm5gepFQWA).
 9. [Hull Robotics Society Discord](https://discord.gg/pYP5kqE).
-10. [Department of Computer Science and Technology Twitter](https://twitter.com/HullCompSci).
-11. [University of Hull Hub](https://discord.gg/WsEy47d9j8)
```
Comment (sbrl) [inline README.md:81]: Why remove this?

File: README.md [hunk 2]
```diff
@@ -164,6 +166,9 @@ Freeside welcome contributions to these lists from staff and students at the Uni
 24. [University of Hull - Hull's digital repository, Hydra](https://hydra.hull.ac.uk/)
 25. [Visualisation and Audibilization of Sorting Algorithms](https://panthema.net/2013/sound-of-sorting/) - Visualize different sorting algorithms and listen the sound of sorting.
 26. [Sentdex Youtube Channel](https://www.youtube.com/c/sentdex) - Youtube channel to learn Machine learning, Deep Learning, Natural Language Processing and python.
+27. [Forage](https://www.theforage.com/) - Offers 5-6 hours virtual work experience programs.
```
Comment (sbrl) [inline README.md]: Looks useful, but not sure this belongs in this section?

**Review by sbrl — COMMENTED** [review]
Thanks for the edits! I've left some comments below for you to review.

File: README.md (hunk 1 above)
Comment (Ki-er) [reply README.md:81]: I swapped them around to keep all discord links together, both are still present within the list

image

File: README.md (hunk 2 above)
Comment (Ki-er) [reply README.md]: Wasnt sure where to pop them, I can move or remove if needed

Comment (CrimsonTome) [reply README.md]: If a new section is to be added, it'd be worth having multiple resources for it ready, for now I'd say it is fine where it is or in business/career

**Comment by CrimsonTome** [comment]
Thank you @KieranRobson for the contributions!

**Pull request merged**